lib/pages/add_bridge/add_bridge.dart

```diff
@@ -240,36 +240,44 @@ class BotController extends State<AddBridge> {
     return ConnectionStatus.error;
   }
 
-  Future<void> fetchLoginFlows(SocialNetwork network) async {
-    final accessToken = client.accessToken;
-    final userId = client.userID;
-    final url = '/${network.apiPath}/provision/v3/login/flows?user_id=$userId';
-
-    final response = await dio.get(
-      url,
-      options: Options(
-        headers: {'Authorization': 'Bearer $accessToken'},
-      ),
-    );
-
-    if (response.statusCode == 200) {
-      final responseJson = response.data;
-      final flows = responseJson['flows'];
-
-      if (flows != null) {
-        print('Available login flows for ${network.name}:');
-        for (var flow in flows) {
-          print('Name: ${flow['name']}, Description: ${flow['description']}, ${flow['id']}');
-        }
-      } else {
-        print('No login flows found for ${network.name}.');
-      }
-    } else if (response.statusCode == 401) {
-      print('Invalid token for ${network.name}.');
-    } else {
-      print('Unexpected error: ${response.statusCode}');
-    }
-  }
+  // Future<void> fetchLoginFlows(SocialNetwork network) async {
```

Comment:
why ?

Reply:
I don't use it in code, but I've done it anyway just in case.
I think I'll use it for future bridges when there are several connection methods or to automate cases where method names change in the future
I can delete this function if you like

Reply:
sounds good, thank you for the explanation !